README.md

```diff
@@ -25,7 +25,7 @@ Prerequisites:
 - [JDK](https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot) 11+
 - [Node](https://nodejs.org) 16+
 - [Python](https://www.python.org/) 3.10+
-  - [Poetry](https://python-poetry.org/)
+  - [Poetry](https://python-poetry.org/): `poetry install`
```

Comment:
`poetry install` will install this project's dependencies (if run in the proper folder), I'd clarify this in the README